lib/dradis-qualys.rb

```diff
@@ -6,5 +6,7 @@
 
 # Load supporting Qualys classes
 require 'qualys/element'
+require 'qualys/asset/evidence'
+require 'qualys/asset/vulnerability'
```

Comment:
:alpha (drink!)

Reply:
@brianp alpha doesn't work here unfortunately. The asset models are using `Qualys.cleanup_html` which is defined in the `qualys/element`. That means the asset requirement has to be done below `qualys/element` otherwise it's going to throw uninitialized constant errors.